data/free worlds intro.txt

```diff
@@ -1537,9 +1613,13 @@ mission "FW Escort 3"
 		
 			`	"Glad to hear it," he says. "The crew of that freighter is depending on you. I know I don't have to tell you what will happen to them if the pirates board them before you get there."`
 				accept
+				
+	on decline
+		log `Did a few escort jobs for the Free Worlds and one of their officers, Tomek, asked me to do a special combat mission for him. Combat is something they should be asking their militia to do; they signed up for that, I didn't.`
```

Comment:
I'd say before there were multiple ships defending the merchants, but here it's just you.